pkg/api/tmpl/chi-interface.tmpl

```diff
@@ -0,0 +1,7 @@
+// ServerInterface represents all server handlers.
```

Comment:
Add a github link to the file you replaced here, so we can keep track of changes as the original template changes.